### ISSUE TITLE
Screen reader capability

### ISSUE THREAD

**Issue by msrichardson** [conveyal/trimet-mod-otp#139]
Please ensure application is able to be used with a screen reader (heuristic analysis found some buttons were missing appropriate labels).
